lib/open-plus.coffee

```diff
@@ -136,9 +136,21 @@ module.exports =
       filename = path.resolve absolute, file
 
     if not fs.existsSync filename
+      filenameExtension = path.extname filename
+      currentFileExtension = path.extname editor.getPath()
+
       # if no extension there, attach extension of current file
-      if not path.extname filename
-        filename += path.extname editor.getPath()
+      if not filenameExtension
+        # if there is no extension, and it is a sass file
+        if currentFileExtension == '.scss'
+          filenameFile = filename.substring(filename.lastIndexOf('/') + 1, filename.length)
+          # if the filename does not already start with _ ... add the underscore.
+          # Fixes https://github.com/klorenz/atom-open-plus/issues/15
+          if filenameFile.substring(0, 1) != '_'
```

Comment:
This assumes ALL scss files start with an "_". I havent used much scss but when I have none of my files started with an "_".